docs/docs/static-vs-normal-queries.md

```diff
@@ -10,27 +10,15 @@ title: Static vs Normal Queries
 >
 > You can help by making a PR to [update this documentation](https://github.com/gatsbyjs/gatsby/issues/14228).
 
-## How StaticQuery differs from page query
+Static queries differ from the normal Gatsby page queries in a number of ways. One example is that Gatsby is capable of handling queries with variables for pages because of it's awareness of pages while generating them. Doing the same for queries inside of specific components is not yet possible, which is where static queries are used. That means data fetched won't be dynamic (hence the name "static" query).
```

Comment:
This is a confusing paragraph. Maybe it would be more clear with some examples?
```suggestion
Static queries differ from Gatsby page queries in a number of ways. One example is that Gatsby is capable of handling queries with variables for pages because of its awareness of page context while generating them. Doing the same for queries inside of specific components is not yet possible, which is where static queries are used. That means data fetched won't be dynamic (hence the name "static" query).
```